src/utils/date.js

```diff
@@ -1,31 +1,27 @@
-function getUTCFullDate(date, separateIntoGroups = "") {
-	if (!date) return "";
-	const dateUTC = date.toUTCString();
+const dynamicDateFormats = {
+	"short date time": 'f',
+	"long date time": 'F',
+	"short date": 'd',
+	"long date": 'D',
+	"short time": 't',
+	"long time": 'T',
+	"relative": 'R'
+};
```

Comment:
Could we make an enum ? I think it would make it easier to use, instead of using sensitive strings. Something like
```js
const DATE_FORMAT = Object.freeze({
    SHORT_DATE_TIME: 'f',
    LONG_DATE_TIME: 'F',
    ...
});
```

Then 
```js
DATE_FORMAT.SHORT_DATE_TIME
```